From: Luca Weiss <luca.weiss@fairphone.com>
Date: Fri, 22 Mar 2024 16:47:32 +0100
Subject: dt-bindings: usb: gpio-sbu-mux: Add Awinic AW35743

The AW35743 DPDT USB Switch[0] can be used for SBU muxing use cases, so
add a compatible for it.

Web page: https://www.awinic.com/en/productDetail/AW35743CSR
Datasheet: https://doc.awinic.com/doc/20230609wm/79681a08-610f-4dc7-ad8d-58f0ab47fa75.pdf

Signed-off-by: Luca Weiss <luca.weiss@fairphone.com>
---
 Documentation/devicetree/bindings/usb/gpio-sbu-mux.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/usb/gpio-sbu-mux.yaml b/Documentation/devicetree/bindings/usb/gpio-sbu-mux.yaml
index d3b2b66..cc7bf34f 100644
--- a/Documentation/devicetree/bindings/usb/gpio-sbu-mux.yaml
+++ b/Documentation/devicetree/bindings/usb/gpio-sbu-mux.yaml
@@ -19,6 +19,7 @@ properties:
   compatible:
     items:
       - enum:
+          - awinic,aw35743
           - nxp,cbdtu02043
           - onnn,fsusb43l10x
           - pericom,pi3usb102
